Gemfile.lock

```diff
@@ -330,3 +330,6 @@ DEPENDENCIES
   uk_postcode (~> 1.0.1)
   unicorn (= 4.8.3)
   webmock (= 1.20.4)
+
+BUNDLED WITH
+   1.10.6
```

Comment:
This is being generated by the version of bundler and ruby on the development VM. If you want I can remove it, but I can only see more and more people adding it as they bundle from within the VM.